app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -99,6 +99,10 @@ class RootActivity : AppCompatActivity(), RootScreen {
         setupDatePicker()
         setupTimePicker()
 
+        createNotificationChannel(this)
```

Comment:
Not all users might want to have reminders, use `IvyFeatures` to make it a customizable setting that's enabled by default